static/app/views/onboarding/targetedOnboarding/components/welcomeBackground.tsx

```diff
@@ -25,45 +23,6 @@ Light.defaultProps = {
   },
 };
 
-const fallingKeyframes = keyframes`
-  0%,  100% {
-    opacity: 1;
-    transform: translateY(0);
-  }
-  30% {
-    transform: translateY(15px);
-  }
-  40% {
-    opacity: 0;
-  }
-  60% {
-    transform: translateY(0);
-  }
-`;
-
-const StarGroup = styled(motion.g)`
-  > g {
-    animation-name: ${fallingKeyframes};
-  }
-`;
-
-StarGroup.defaultProps = {
-  onAnimationEnd: (e: React.AnimationEvent) => {
```

Comment:
Nah, the component wasnt used in this file (nor was it exported)! The only reason why it wasnt marked as unused was because of the StarGroup.defaultProps side effect...